pmaweb/templates/components/_icon-library.html

```diff
@@ -0,0 +1,989 @@
+<svg width="0" height="0" style="display:none;">
```

Comment:
should it be a .svg ?

Reply:
Yes. It's an SVG library (or SVG sprite), that I just call each "layer" when I need it. 
It's more resource-saving because we just have one file instead of a lot of SVG's across the project. It's easier to maintain too.

Since it's an inline SVG, we don't need to pay attention to cross-origin protection too.

You can read more [here](https://www.creativebloq.com/features/the-complete-guide-to-svg/6).